config.py.example

```diff
@@ -0,0 +1,9 @@
+"""The server configuration."""
+
+http_host = '0.0.0.0'  # Use the actual IP address, binding to 0.0.0.0 sucks!
```

Comment:
Pri HTTP v podstate ani nie, pri UDP áno a keď už, chcela som to mať rovnaké :D (Pravdepodobne bude totiž použitý server mať 2 sieťovky, controllery budú samostatný subnet a UDP pôjde do dnu a HTTP do von, takže by som tam rada videla nakonfigurované také.)